Appraisals

```diff
@@ -1,26 +1,15 @@
 # frozen_string_literal: true
 
 RAILS_VERSIONS = %w[
-  4.2.11.3
-  5.1.7
-  5.2.5
-  6.0.3.6
-  6.1.3.1
   7.0.1
```

Comment:
I'm not familiar with Appraisals do I need to add '7.2.0', '7.1.0' here or is it enoght in the .github/workflows/test.yml